api.js

```diff
@@ -25,5 +25,14 @@ const api = axios.create({
     })
   }
 
+  const getCommentsById = (id) => {
+    return api.get(`/articles/${id}/comments`)
+    .then(({data})=> {
+        return data.comments
+    })
+    .catch((err)=>{
```

Comment:
Returning an error here will stop it reaching any catch block elsewhere, it will turn a promise.reject into a promise.resolve but with the error as a value